auto_submit/test/service/config_test.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:auto_submit/service/config.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Config', () {
+    late Config config;
+
+    setUp(() {
+      config = Config();
+    });
+    test('Two repos with tree status', () {
```

Comment:
Make sense to me, but since all the unit tests in config_test are testing const values, I will remove the whole unit test. Is it ok to not having the unit test for config?